.github/workflows/linux.yml

```
@@ -15,13 +15,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        crystal_bootstrap_version: [1.2.2, 1.3.2, 1.4.1, 1.5.1, 1.6.2, 1.7.2]
+        crystal_bootstrap_version: [1.2.2, 1.3.2, 1.4.1, 1.5.1, 1.6.2]
+        env: ["USE_PCRE1=true"]
```

Hash notation seems more natural:

```suggestion
        env: 
          USE_PCRE1: true
```

---

Oh shit, I've noticed just now that's not `jobs.<job_id>.env` but rather a matrix variable called `env`. In this case using hash notation rather won't work, sorry for the mixup. btw, I'd suggest renaming this variable to some more descriptive, like `flags`, so it gives better understanding what is it for and won't mixup with the `env` key, which is _a thing_ for GHA.

---

@straight-shoota This most likely broke the CI :(

---

Oh shit. Thanks for noticing.
What's really, really bad about this is that the GitHub Actions summary on the PR and commit doesn't mention this. The workflow file failed to parse but that's not reported as a failure? WTF.